graphene_django/converter.py

```diff
@@ -133,13 +133,17 @@ def convert_choice_field_to_enum(field, name=None):
 
 
 def convert_django_field_with_choices(
-    field, registry=None, convert_choices_to_enum=True
+    field, registry=None, convert_choices_to_enum=None
 ):
     if registry is not None:
         converted = registry.get_converted_field(field)
         if converted:
             return converted
     choices = getattr(field, "choices", None)
+    if convert_choices_to_enum is None:
```

Comment:
A reason why I moved it into `convert_django_field_with_choices` and not implemented it on `construct_fields` or `DjangoObjectType` is, that `django-graphene-plus` uses `convert_django_field_with_choices` directly and would skip/ignore the setting or would need to manually look it up.
See: https://github.com/0soft/graphene-django-plus/blob/master/graphene_django_plus/input_types.py#L28